app/templates/edit_supplier_name.html

```diff
@@ -0,0 +1,45 @@
+{% from 'macros/page_heading.html' import page_heading %}
+
+{% extends "_base_page.html" %}
+{% block page_title %}
+  Change name – {{ supplier.name }} – Digital Marketplace admin
+{% endblock %}
+
+{% block content %}
+  {%
+    with items = [
+      {
+        "link": url_for('.index'),
+        "label": "Admin home"
+      },
+      {
+        "label": supplier.name
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+  <div class="page-container">
+    {% with heading = "Change supplier name" %}
```

Comment:
It might be nice to have the old supplier name shown somewhere other than in the breadcrumb - it took me a while to spot it there.